Microsoft.DotNet.Interactive/LanguageService/IKernelLanguageService.cs

```diff
@@ -1,11 +1,13 @@
 ﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System.Threading.Tasks;
+using Microsoft.DotNet.Interactive.Commands;
+
 namespace Microsoft.DotNet.Interactive.LanguageService
 {
-    public class HoverParams
+    public interface IKernelLanguageService<TCommand> where TCommand: LanguageServiceCommandBase
```

Comment:
I think it makes sense to generalize this to all commands, e.g. `IKernelCommandHandler<T>`. There's nothing special about language services.